content/post/36-a-look-at-windows.md

```diff
@@ -0,0 +1,52 @@
+---
+date: 1970-01-01
+url: 36-a-look-at-windows
+title: A look at go-ipfs on Windows
+author: Dominic Della Valle
+---
+
+`go-ipfs` is built on top of [Golang](https://golang.org/), which allows for some cross platform compatibility, however it doesn't cover everything. Let's take a look at some of the issues of running `go-ipfs` on Windows and the upcoming solutions for them.
+
+### Log output
+![colour](img/colour.png)  
```

Comment:
It might be helpful to add a caption just to clarify the top screenshot is what output used to look like and the bottom is what it looks like now. Or split them up and put one before each of these paragraphs.